editor/src/components/navigator/left-pane.tsx

```diff
@@ -763,6 +768,249 @@ const GithubSpinner = () => {
   )
 }
 
+interface RepositoryRowProps extends RepositoryEntry {
+  importPermitted: boolean
+}
+
+const RepositoryRow = (props: RepositoryRowProps) => {
+  const colorTheme = useColorTheme()
+
+  const dispatch = useEditorState((store) => store.dispatch, 'RepositoryRow dispatch')
+
+  const projectID = useEditorState((store) => store.editor.id, 'RepositoryRow projectID')
+
+  const importRepository = React.useCallback(() => {
+    const parsedTargetRepository = parseGithubProjectString(props.fullName)
+    if (parsedTargetRepository == null) {
+      dispatch(
+        [
+          showToast(
+            notice(
+              `Error when attempting to import a repository with repo: ${props.fullName}`,
+              'ERROR',
+            ),
+          ),
+        ],
+        'everyone',
+      )
+    } else {
+      void getBranchContent(
+        dispatch,
+        parsedTargetRepository,
+        forceNotNull('Should have a project ID.', projectID),
+        props.defaultBranch,
+      )
+    }
+  }, [dispatch, projectID, props.fullName, props.defaultBranch])
+
+  return (
+    <div
+      style={{
+        minHeight: 40,
+        display: 'flex',
+        flexDirection: 'row',
+        gap: 8,
+        alignItems: 'center',
+        paddingLeft: 8,
+        paddingRight: 8,
+        paddingBottom: 8,
+        paddingTop: 8,
+        borderBottom: '1px solid #ccc',
+      }}
+    >
+      <div
+        style={{
+          borderRadius: '50%',
+          width: 20,
+          height: 20,
+          border: '1px solid #ccc',
+          backgroundImage: `url(${props.avatarUrl})`,
+          backgroundPosition: 'center',
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat',
+        }}
+      />
+      <div style={{ flexGrow: 1 }}>
+        <span style={{ fontWeight: 600, textOverflow: 'ellipsis' }}>
+          {props.name ?? props.fullName}
+        </span>{' '}
+        <br />
+        <span style={{ opacity: 0.5 }}>
+          {props.private ? 'private' : 'public'} · <TimeAgo date={props.updatedAt} />
```

Comment:
when a repository is not found, `private` is shown and the `TimeAgo` will start going up from `0` every second. e.g.:
<img width="269" alt="Screen Shot 2022-10-31 at 4 48 48 PM" src="https://user-images.githubusercontent.com/1081051/199051210-8a3d0969-d0a2-4dd7-b71f-bce09cd6edaa.png"/> because of how the `additionalEntry` item is added.

Reply:
note: with the current implementation it's impossible to see (and so import) public repos which are not owned by the GH user. Imho it would be nice to be able to do this (but of course `Save to Github` should be disabled if write access is not available.

Reply:
Yeah, I think it's going to be better to hide the time for that fake entry.

Reply:
Since this is for "connecting" to a repo, it makes more sense that these are repositories that you can modify.

Reply:
Fixed in 1a05bc536.